packages/babel-helper-create-class-features-plugin/src/index.ts

```diff
@@ -117,7 +123,7 @@ export function createClassFeaturePlugin({
             const getName = `get ${name}`;
             const setName = `set ${name}`;
 
-            if (path.node.kind === "get") {
+            if (path.isClassPrivateMethod() && path.node.kind === "get") {
```

Comment:
nit: we can avoid to call `path.isClassPrivateMethod` twice using another outer `if`